lib/model/app_model/list_invoices.dart

```diff
@@ -2,18 +2,24 @@ import 'package:cln_common/cln_common.dart';
 import 'package:clnapp/model/app_model/app_utils.dart';
 
 class AppListInvoices {
-  List<AppInvoice> invoice;
+  List<AppInvoice?> invoice;
 
   AppListInvoices({this.invoice = const []});
 
   factory AppListInvoices.fromJSON(Map<String, dynamic> json,
-      {bool snackCase = false, bool isObject = false}) {
+      {bool snackCase = false, bool isObject = false, required String status}) {
```

Comment:
```suggestion
      {bool snackCase = false, bool isObject = false, required String status}) {
```

why this required String?